### PR TITLE
Fix RCE with arguments passed to pygmentize

### PR DESCRIPTION
**Pull request by h4ckninja**

### CODE REVIEW

File: src/Pygmentize/Pygmentize.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public static function highlight($source, $language, $encoding = "utf-8", $forma
       else
         $args = sprintf(" -f %s -g -O encoding=%s,style=%s,lineos=1", $formatter, $encoding, $style);
 
-      $proc = proc_open(self::PIGMENTS_BINARY.$args, $dspec, $pipes);
+      $proc = proc_open(self::PIGMENTS_BINARY.escapeshellcmd($args), $dspec, $pipes);
```
Comment (cebe) [inline src/Pygmentize/Pygmentize.php:53]: Input should be escaped by `escapeshellarg` instead and that should be applied to each argument separately in line 49 and 51.

See:

> http://php.net/manual/en/function.escapeshellcmd.php 
> Warning
> escapeshellcmd() should be used on the whole command string, and it still allows the attacker to pass arbitrary 
> number of arguments. For escaping a single argument escapeshellarg() should be used instead.

and http://php.net/manual/en/function.escapeshellarg.php

Comment (dedalozzo) [reply src/Pygmentize/Pygmentize.php:53]: I guess @cebe is right. I will fix this.

Comment (h4ckninja) [reply src/Pygmentize/Pygmentize.php:53]: Thanks @cebe for the assist. :)

**Comment by h4ckninja** [comment]
Closing to avoid confusion. @cebe has a much better fix in #3.

**Pull request closed without merging**